crates/ruff_python_resolver/src/python_platform.rs

```diff
@@ -5,3 +5,16 @@ pub(crate) enum PythonPlatform {
     Linux,
     Windows,
 }
+
+impl PythonPlatform {
+    /// Returns the platform-specific library names. These are the candidate names for the top-level
+    /// subdirectory within a virtual environment that contains the `site-packages` directory
+    /// (with a `pythonX.Y` directory in-between).
+    pub(crate) fn lib_names(&self) -> &[&'static str] {
+        match self {
+            PythonPlatform::Darwin => &["lib"],
+            PythonPlatform::Linux => &["lib", "lib64"],
```

Comment:
`lib64` is something that i could totally see some distro using but i've never seen in practice so it's hard to test